libpod/options.go

```diff
@@ -2355,6 +2356,20 @@ func WithVolatile() CtrCreateOption {
 		}
 
 		ctr.config.Volatile = true
+
+		return nil
+	}
+}
+
+// WithInfraUserns sets the userns for the infra container in a pod.
+func WithInfraUserns(userns specgen.Namespace) PodCreateOption {
```

Comment:
Suggest renaming to "WithPodUserNS(...)`

Reply:
Fixed